lib/cli.js

```diff
@@ -14,6 +14,10 @@ const runFunction = (name, func, parallel) => {
 }
 
 const executeTask = (name, chainValue, taskArgs, parallel, reportFunction) => {
+  if (!taskDefinition.tasks) {
+    return Promise.resolve();
```

Comment:
Could you please add a error-log message here, that the tasks are not yet initialized.

Reply:
of course. Error log has been added